### ISSUE TITLE
Populate the project spreadsheet

### ISSUE THREAD

**Issue by volkanunsal** [BetaNYC/betanyc-projects-list#48]
This is for you Noel. You know what to do. Please make it so.

**Comment by volkanunsal** [comment]
One thing to note, please make sure to prefix all categories –– except for primary categories –– with their name, followed by a colon. Example: `showcase:Hacker Tools`. Whatever follows this prefix will be the name of that category, so make sure this looks like a normal category name. No dashes or underscores.

**Comment by fkh** [comment]
@volkanunsal what is the URL to the projects spreadsheet, and can I have edit rights? 

I want to use the list of project urls from the feed at http://projects.beta.nyc/projects, it has a few gaps at the moment.

**Comment by volkanunsal** [comment]
@fkh 

It's here

http://betanyc.github.io/betanyc-projects-list/#/

But it's not completely done yet. We're going to have the data up at this spreadsheet. I can give you edit rights.

https://docs.google.com/spreadsheets/d/1SkuQ6kbZEycCEFeDKwnWx4PLc5P1uHqQMA7LQcfPHYE/edit#gid=0

**Comment by volkanunsal** [comment]
I made it editable by all.

**Comment by fkh** [comment]
thanks! and the project list now gets published at http://betanyc-api.herokuapp.com/api/projects ?

**Comment by volkanunsal** [comment]
Yes, but that feed doesn't have the items from the spreadsheet at the moment. We want to get the taxonomy right first.

**Comment by fkh** [comment]
@volkanunsal where is the most current feed of projects? i am using http://projects.betanyc.us/projects right now, should I switch to something else? 

I'm using it for this -- https://github.com/fkh/dotgov-screenshots
